impeller/renderer/render_target.cc

```diff
@@ -351,10 +349,7 @@ RenderTarget RenderTargetAllocator::CreateOffscreenMSAA(
       return {};
     }
   }
-#ifdef IMPELLER_DEBUG
```

Comment:
I removed this IFDEFS because the texture setlabel implemenations have the ifdef and we're not string formatting here anymore.